src/miniscript/types/mod.rs

```diff
@@ -14,12 +14,15 @@
 
 pub mod correctness;
 pub mod malleability;
+pub mod extra_props;
 
 use std::{error, fmt};
 
-use miniscript::astelem::AstElem;
+use miniscript::decode::Terminal;
 pub use self::correctness::{Correctness, Base, Input};
 pub use self::malleability::{Dissat, Malleability};
+pub use self::extra_props::ExtData;
```

Comment:
`extra_props` is not added til a later commit, so this commit doesn't compile

Reply:
@apoelstra , 
I don't think it's easy to create a perfect bijection between taking strings as test inputs and changing the existing tests from Miniscript types.
 
The only reason we required test string types inputs was because typing in tests for Mini
script nodes using Miniscript{node: astelem, ty:type, ext: ext} was really hard.

Reply:
I can however make a commit which removes the tests which relied on this. And then readd with the string inputs